tests/unit/OptionsListUtilsTest.js

```diff
@@ -597,23 +619,41 @@ describe('OptionsListUtils', () => {
                 // When we call getSidebarOptions() with no search value
-                const results = OptionsListUtils.getSidebarOptions(REPORTS, PERSONAL_DETAILS, 0, CONST.PRIORITY_MODE.GSD);
+                const results = OptionsListUtils.getSidebarOptions(REPORTS_WITH_MORE_PINS, PERSONAL_DETAILS, 0, CONST.PRIORITY_MODE.GSD);
 
                 // Then expect all of the reports to be shown both multiple and single participant except the
                 // report that has no lastMessageTimestamp and the chat with Thor who's message is read
-                expect(results.recentReports.length).toBe(_.size(REPORTS) - 2);
+                expect(results.recentReports.length).toBe(_.size(REPORTS_WITH_MORE_PINS) - 2);
 
                 // That no personalDetails are shown
                 expect(results.personalDetails.length).toBe(0);
 
-                // And Mister Fantastic is alphabetically the fourth report and has an unread message
-                // despite being pinned
-                expect(results.recentReports[4].login).toBe('reedrichards@expensify.com');
 
-                // And Black Panther is alphabetically the first report and has an unread message
-                expect(results.recentReports[0].login).toBe('tchalla@expensify.com');
+                //------------------- Pinned report---------
+                // Pinned reports always on top regardless of having unread message or not  
+                // and regardless having IOU debt or not
+                // ordered by alphabetical order. 
+                // D report name (Alphabetically first among pinned tabs) 
+                expect(results.recentReports[0].login).toBe('d_email@email.com');
+        
+                // Mister Fantastic report name (Alphabetically second among pinned tabs) 
+                expect(results.recentReports[1].login).toBe('reedrichards@expensify.com');
+
+                // Z report name (Alphabetically third among pinned tabs) 
+                expect(results.recentReports[2].login).toBe('z_email@email.com');
+
+                //------------ Unpinned reports ------------
+                // Unpinned report name ordered alphabetically after pinned report
+                //Black Panther report name
+                expect(results.recentReports[3].login).toBe('tchalla@expensify.com');
+
+                //Captain America report name
+                expect(results.recentReports[4].login).toBe('steverogers@expensify.com');
+
+                //Invisible woman report name. Has an IOU debt.
+                expect(results.recentReports[5].login).toBe('suestorm@expensify.com');
 
-                // And Mister Sinister is alphabetically the fifth report and has an IOU debt despite not being pinned
-                expect(results.recentReports[5].login).toBe('mistersinister@marauders.com');
+                //Spider-Man report name is last report
+                expect(results.recentReports[8].login).toBe('peterparker@expensify.com');
```

Comment:
Does seem like a good option to remove existing tests. Could you modify the IOU report test instead of removing it?

Reply:
okay